### DISCUSSION TITLE
Скрипт по МегаМаркету

### DISCUSSION THREAD

**Discussion by TimurGumHub** [xob0t/MM-tools#5] [Q&A]
Здравствуйте! Я следовал вашим шагам для установки двух расширений чтобы можно было сортировать товары на Мегамаркете. И у меня получается только видеть обновленную цену когда я захожу на страницу самого товара. Но не получается увидеть множество товаров отсортированных  с обновленными ценами. Подскажите пожалуйста, в чем может быть дело?

**Comment by xob0t** [comment]
на сайте верстка изменилась, надо скрпит обновить

**Reply by xob0t** [reply]
готово
если не обновится сам, переустановите Megamarket Price Calculator

**Reply by TimurGumHub** [reply]
Спасибо большое! Все работает)

**Reply by Pavel239Pavel** [reply]
Здравствуйте. Опять обновилась видимо вёрстка, работает только на странице карточки товара.

**Reply by xob0t** [reply]
чуть подлатал, но его по хорошему нужно переписывать, а то сейчас на костылях работает
